machines/bleShare/scan/scanMachine.ts

```diff
@@ -75,6 +77,7 @@ export const scanMachine =
           },
         },
         checkStorage: {
+          entry: 'setOpenId4VPRef',
```

Comment:
why this is set as entry but not in isOnlineSharing guard https://github.com/mosip/inji-wallet/pull/1628/files#diff-dd4fd1a72e521e80bc2f4e724ea91d3f67e6e8afae91f62a5dd196d4681cdbbfR345